ouroboros-network/src/Ouroboros/Network/NodeToClient.hs

```diff
@@ -584,6 +586,16 @@ networkErrorPolicies = ErrorPolicies
       , ErrorPolicy
           $ \(_ :: IOManagerError)
                 -> Just Throw
+
+        -- Using 'error' throws.
+      , ErrorPolicy
+          $ \(_ :: ErrorCall)
+                -> Just Throw
+
+        -- Using panic throws.
```

Comment:
For consistency with the case above, I would:
```suggestion
        -- Using 'panic' throws.
```

Reply:
Done, thanks.